src/python/tests/database/test_core_server.py

```diff
@@ -0,0 +1,73 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of :mod:`ensembl.io.genomio.gff3.process_gff3` module.
+
+"""
+
+from typing import TypedDict
+from unittest.mock import Mock, patch
```

Comment:
There's an alternative to do this without adding this import (`mocker` is already part of `pytest`), but will need some time to collect the information to put it here.